### PR TITLE
add ReadMe

### PR DESCRIPTION
**Pull request by sarahizar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
-# Recipe-Nutri
```
Comment (AnnaT7459) [inline README.md:1]: This is written twice if you want to delete one

**Comment by sarahizar** [comment]
I can do that when i get off!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+## Table of Contents
+-Description
+-Features
+-Usage
+-Technologies
+-Screenshot
+-Link
+
+## Description
+RecipeNutri is a web application that allows users to find and explore recipes while providing detailed nutritional information. Users can search for recipes by recipe name, view cooking instructions, and get accurate nutrition facts for their chosen dishes. This README provides an overview of the application, instructions for usage, and details for developers.
```
Comment (AnnaT7459) [inline README.md]: In the description, we need to include the user story, but from our point of view to include our motivations and what we learned. I use the acceptance criteria to help me write this part. 

Project Requirements
You and your group will use everything you’ve learned over the past six weeks to create a real-world front-end application that you’ll be able to showcase to potential employers. The user story and acceptance criteria will depend on the project that you create, but your project must fulfil the following requirements:

Use a CSS framework other than Bootstrap.

Be deployed to GitHub Pages.

Be interactive (i.e: accept and respond to user input).

Use at least two server-side APIs.

Does not use alerts, confirms, or prompts (use modals).

Use client-side storage to store persistent data.

Be responsive.

Have a polished UI.

Have a clean repository that meets quality coding standards (file structure, naming conventions, follows best practices for class/id-naming conventions, indentation, quality comments, etc.).

Have a quality README (with unique name, description, technologies used, screenshot, and link to deployed application).

Finally, You must add your project to the portfolio that you created in Module 2. ***WE NEED TO ALL DO THIS - WHOOPS!***

File: README.md [hunk 3]
```diff
@@ -1 +1,55 @@
...
+URL: https://api.nal.usda.gov/fdc/v1/foods/search
+Access Date: [10/24/2023]
+
+**The MealDB API Reference**
+Title of the API: The Meal DB
+Author or Organization: TheMealDB
+URL: https://www.themealdb.com/api/json/v1/1/search.php?s=
+Access Date: [10/24/2023]
+
+
```
Comment (AnnaT7459) [inline README.md]: We also need to include credits for both of the APIs in APA format as well as personal communication credits for Colin, Josh, and Max for assisting us.

Comment (AnnaT7459) [reply README.md]: Example for APIs: 
OpenWeather. (2023). 5-Day Weather Forecast API Documentation. https://openweathermap.org/forecast5

Example for personal communication: 
H. Burke (personal communication, October 15, 2023) A. Warren (personal communication, October 9, 2023) J. Diehl TA (personal communication, October 10, 2023) M. Mayfield (personal communication, October 12, 2023)

**Review by AnnaT7459 — CHANGES_REQUESTED** [review]
See specific comments in README

**Review by AnnaT7459 — APPROVED** [review]
Thanks for updating! Looks good!

**Review by natemcmahon — APPROVED** [review]

**Pull request merged**